src/common/components/LoadingSpinner/WithLoadingOverlay.tsx

```diff
@@ -1,6 +1,17 @@
 import { CenteredSpinnerContainer, DimmableContent, NoContent } from 'common/styles/utilities';
 import { FC, PropsWithChildren, useEffect, useState } from 'react';
 import { LoadingSpinner } from './LoadingSpinner';
+import styled from 'styled-components';
+
+const StyledLoadingSpinner = styled.div`
+  padding: 3rem 0rem;
+  text-align: center;
+  
+  #loading-spinner {
+    width: 4em;
+    height: 4em;
+  }
+`;
```

Comment:
I don't think we should be setting an absolute size for the spinner in `WithLoadingOverlay`. Since you are setting loading spinner to an absolute size, it makes `WithLoadingOverlay` less flexible. Instead what do you think of just styling it in the Profile Picture wrapper (which it looks like you've already done)

The padding stuff may affect this as well.

Reply:
Agreed. Your suggested approach makes sense to me.

Reply:
On second thought, I realized that I still needed to set the spinner's size for the WithLoadingOverlay component. I changed the above code and made it so that the width and height of the spinner can be optionally set. By default, the width and height is set to 4em. What do you think? Someone could make the argument that we should use the WithLoadingOverlay component in both places. However, when the profile picture is updating, the spinner is shown on top of the previous profile picture without a dimming effect. So, at the moment, the two experiences are different.